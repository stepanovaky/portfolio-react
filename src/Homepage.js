import React from 'react';
import KingQuizApp from './Projects/KingQuizApp';

function Homepage() {
    return(
        <div className="homepage">
            <KingQuizApp />
        </div>
    )
}

export default Homepage;