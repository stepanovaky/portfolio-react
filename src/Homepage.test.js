import React from 'react';
import renderer from 'react-test-renderer';
import Homepage from './Homepage';

test('it renders correctly', () => {
    const tree = renderer
        .create(<Homepage />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})