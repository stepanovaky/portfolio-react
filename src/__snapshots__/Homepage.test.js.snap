// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="homepage"
>
  <p>
    import one project you're most proud of
  </p>
</div>
`;
